src/WatchDogs.Persistence.EntityFramework/DataInserter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WatchDogs.Contracts;
+
+namespace WatchDogs.Persistence.EntityFramework;
+public class DataInserter
```

Comment:
Why no interface? This creates tight coupling.

Reply:
I was so exited that code is working that I forgot